larray/core/array.py

```diff
@@ -5631,7 +5631,7 @@ def _binop(opname):
         fullname = f'__{opname}__'
         super_method = getattr(np.ndarray, fullname)
 
-        def opmethod(self, other):
+        def opmethod(self, other: Union[Scalar, 'Array']) -> 'Array':
```

Comment:
OK. But I think it could be useful (for me) to add type hints to methods which I don't fully understand how they (are supposed to) behave (`getitem`, `_binop`, all that private functions/methods linked to `getitem` and `aggregate`).

Reply:
Makes sense. Ok for arguments of functions (whether internal or not) but not in this PR, please. Also, I still find that typing **internal** attributes is not worth the added maintenance burden and decrease in readability.